src/style/blocks/card.css

```diff
@@ -0,0 +1,80 @@
+.ProductCard {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 199px;
+  padding: 32px 16px 0;
+}
+
+.ProductCard__image {
+  width: 160px;
+  height: 134px;
+  margin: auto;
+  padding-bottom: 40px;
+}
+
+.ProductCard__Header {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  margin-bottom: 4px;
+}
+
+.ProductCard__GoodsCode {
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.ProductCard__Rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.ProductCard__Reviews {
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.ProductCard__PriceContainer {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.ProductCard__Price {
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.ProductCard__Count {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.ProductCard__ButtonBuy {
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #fff;
+  background: #00acdc;
+  border-radius: 5px;
+  padding: 12px 0;
+  margin-bottom: 16px;
+  text-decoration: none;
+}
```

Comment:
Add a transition property, it will look better with it
transition:  background-color 0.3ms; (smth like this)

Reply:
agree, transition should be here